game/src/cutscene.rs

```diff
@@ -219,32 +216,26 @@ fn make_panel(
                 Layout::BossSpeaking => Widget::custom_row(vec![
                     Widget::draw_batch(
                         ctx,
-                        GeomBatch::screenspace_svg(
-                            ctx.prerender,
-                            "system/assets/characters/boss.svg",
-                        )
-                        .scale(0.75)
-                        .autocrop(),
+                        GeomBatch::load_svg(ctx.prerender, "system/assets/characters/boss.svg")
+                            .scale(0.75)
+                            .autocrop(),
                     ),
                     scenes[idx].msg.clone().wrap_to_pct(ctx, 30).draw(ctx),
                     Widget::draw_svg(ctx, "system/assets/characters/player.svg").align_right(),
                 ]),
                 Layout::Extra(name, scale) => Widget::custom_row(vec![
                     Widget::draw_batch(
                         ctx,
-                        GeomBatch::screenspace_svg(
-                            ctx.prerender,
-                            "system/assets/characters/boss.svg",
-                        )
-                        .scale(0.75)
-                        .autocrop(),
+                        GeomBatch::load_svg(ctx.prerender, "system/assets/characters/boss.svg")
+                            .scale(0.75)
+                            .autocrop(),
                     ),
                     Widget::col(vec![
                         Widget::draw_batch(
                             ctx,
-                            GeomBatch::screenspace_svg(
+                            GeomBatch::load_svg(
                                 ctx.prerender,
-                                format!("system/assets/characters/{}.svg", name),
+                                &format!("system/assets/characters/{}.svg", name),
```

Comment:
If there were many of these calls, I'd vote for doing `<I: Into<String>>` for `load_svg`, but I only see one. It's an easy refactor to make later if these creep through.